plasmapy/diagnostics/charged_particle_radiography.py

```diff
@@ -1115,118 +1260,144 @@ def max_deflection(self):
 
         return max_deflection * u.rad
 
-    # *************************************************************************
-    # Synthetic diagnostic methods (creating output)
-    # *************************************************************************
-
-    def synthetic_radiograph(
-        self, size=None, bins=[200, 200], ignore_grid=False, optical_density=False
-    ):
+    def _enforce_order(self):
         r"""
-        Calculate a "synthetic radiograph" (particle count histogram in the
-        image plane).
 
-        Parameters
-        ----------
-        size : `~astropy.units.Quantity`, shape (2,2)
-            The size of the detector array, specified as the minimum
-            and maximum values included in both the horizontal and vertical
-            directions in the detector plane coordinates. Shape is
-            [[hmin,hmax], [vmin, vmax]]. Units must be convertable to meters.
+        The Tracker methods could give strange results if setup methods
+        are used again after the simulation has run. This method
+        raises an error if the simulation has already been run.
 
-        bins : array of integers, shape (2)
-            The number of bins in each direction in the format [hbins, vbins].
-            The default is [200,200].
+        """
 
-        ignore_grid: bool
-            If True, returns the intensity in the image plane in the absence
-            of simulated fields.
+        if self._has_run:
+            raise RuntimeError(
+                "Adding new particles or meshes after a simulation "
+                "is complete is not supported. Create a new simulation "
+                "object to begin a new run."
```

Comment:
```suggestion
                "Modifying the `Tracker` object after running the "
                "simulation is not supported. Create a new `Tracker` "
                "object for a new simulation."
```

Generalize the error statement a little more so it's more encompassing of possible future modifications and, if so, would not need rewording.